### PR TITLE
Scott Feedback #1

### PR DESCRIPTION
**Pull request by Cold-A-Muse**
## Proposed Changes
Changes proposed by @defi-er at https://airtable.com/tblnbVhCBsgswbUMH/viwNxew66TXq1ttxf?blocks=hide

### CODE REVIEW

File: packages/web/src/pages/index.tsx [hunk 1]
```diff
@@ -31,16 +31,16 @@ const Index = (_: Props) => {
   }, [router])
 
   return (
-    <>
+    <div style={{ background: 'white' }}>
```
Comment (hhatto) [inline packages/web/src/pages/index.tsx:34]: [nits] Does this background color setting need to be reflected on all pages?

**Review by hhatto — APPROVED** [review]
I just commented on one point.
I don't have a problem with the implementation, so I think it's fine to merge it as long as it looks OK.

File: packages/web/src/pages/index.tsx (hunk 1 above)
Comment (aggre) [reply packages/web/src/pages/index.tsx:34]: That certainly looks like unnecessary styles...

But I guess it's all good!

**Comment by aggre** [comment]
@Cold-A-Muse Thanks good tweaks!

File: packages/web/src/pages/index.tsx (hunk 1 above)
Comment (defi-er) [reply packages/web/src/pages/index.tsx:34]: The white background was chosen for a few reasons:

-Gives a modern aesthetic to the UI
-Maximum contrast with text 
-More freedom to use different colors

Here are other examples of crypto websites opting for white backgrounds:
[](https://ourzora.com)
[](https://opensea.io)

White offers a better canvas for layers with shadows, text, images with drop shadows which Stakes Social has plenty of.

**Pull request merged**